sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_http_logging_policy.py

```diff
@@ -76,15 +76,15 @@ def on_request(self, request: PipelineRequest[HTTPRequestType]) -> None:
     def on_response(
         self,
         request: PipelineRequest[HTTPRequestType],
-        response: PipelineResponse[HTTPRequestType, HTTPResponseType],
+        response: PipelineResponse[HTTPRequestType, HTTPResponseType],  # type: ignore[override]
```

Comment:
This one seems to be impossible to type correctly without importing from private azure-core namespaces, because the underlying implementation is typed with non-public objects.